src/components/questions/handle-key-press.property.spec.ts

```
@@ -0,0 +1,360 @@
+          "tabPrev",
+          "tabCreate",
+          "arrowDownMid",
+          "arrowUpMid",
+          "tabShiftNone",
+          "tabNoneAtEnd",
+          "backspaceNone",
+        ),
+        (mode) => {
+          const t = (v: string, sel: number) => createTextarea(v, sel);
+
+          const base: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)